lakefile.toml

```diff
@@ -0,0 +1,21 @@
+name = "formal_conjectures"
+keywords = ["math"]
+defaultTargets = ["FormalConjectures"]
+
+[leanOptions]
+# Pretty-prints `fun a ↦ b`
+pp.unicode.fun = true
+# Don't assume a typo is a new variable
+autoImplicit = false
+relaxedAutoImplicit = false
+# Switch off warnings generated by `sorry`
+warn.sorry = false
```

Comment:
would it be easy to keep `warn.sorry = true` , but only for the ForMathlib directory?

Reply:
Hmm, I don't think so. An alternative would be for the `category` attribute to suppress `sorry` warnings, but I am not sure this is possible either